cfme/tests/perf/test_performance.py

```diff
@@ -0,0 +1,36 @@
+from timeit import timeit
+
+import pytest
+
+from cfme.base.ui import navigate_to
+from cfme.services.myservice import MyService
+from cfme.tests.test_db_migrate import download_and_migrate_db
+from cfme.utils.conf import cfme_data
+
+
+@pytest.fixture
+def appliance_with_performance_db(temp_appliance_extended_db):
+    app = temp_appliance_extended_db
+    db_backups = cfme_data['db_backups']
+    performance_db = db_backups['performance_510']
+    download_and_migrate_db(app, performance_db.url, performance_db.desc)
+    yield app
+
+
+def test_services_performance(appliance_with_performance_db):
```

Comment:
need `test_requirements` marker

Reply:
@mshriver @digitronik I don't really know what requirement should I link this test with.

Reply:
@jaryn I would link this test to specific Focus Area requirement like if this test belongs to `Service FA` then link to `service`.
```
@test_requirements.service
def test_services_performance(appliance_with_performance_db):
```

If we don't have test_requinment for FA; please feel free to create a new one.